Task01/Task01/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Task01
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.ForegroundColor = ConsoleColor.Green;
+            Console.WriteLine("Костер");
+            Console.ForegroundColor = ConsoleColor.Red;
+            Console.WriteLine("Зинаида Гиппиус");
+            Console.ForegroundColor = ConsoleColor.DarkYellow;
+            Console.WriteLine("Живые взоры я встречаю...\r\nОгня, огня! Костер готов.\r\nЯ к ближним руки простираю,\r\nЯ жду движенья, знака, слов...\r\n\r\nС какою радостною мукой\r\nВ очах людей ловлю я свет!\r\nНо говорю... и дышит скукой\r\nИх утомительный ответ.\r\n\r\nЯ отступаю, безоружный,\r\nИ длю я праздный разговор,\r\nИ лью я воду на ненужный,\r\nНа мой безогненный костер.\r\n\r\nО, как понять, что это значит?\r\nКого осудим, – их? меня?\r\nДуша обманутая плачет...\r\nКостер готов – и нет меня.");
```

Comment:
Если вы хотите задать текст из нескольких строк как строковую константу, излишне сначала возвращать точку вывода в начало строки(`\r`). Достаточно сразу перейти на новую строку (`\n`).
```suggestion
            Console.WriteLine("Живые взоры я встречаю...\nОгня, огня! Костер готов.\nЯ к ближним руки простираю,\r\nЯ жду движенья, знака, слов...\n\nС какою радостною мукой\r\nВ очах людей ловлю я свет!\nНо говорю... и дышит скукой\nИх утомительный ответ.\n\nЯ отступаю, безоружный,\nИ длю я праздный разговор,\nИ лью я воду на ненужный,\nНа мой безогненный костер.\n\nО, как понять, что это значит?\nКого осудим, – их? меня?\nДуша обманутая плачет...\nКостер готов – и нет меня.");
```

Reply:
Поняла, исправлено.